iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/content/HiveBasedDocumentContentUrlImporterJobTest.java

```diff
@@ -0,0 +1,128 @@
+package eu.dnetlib.iis.wf.importer.content;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import eu.dnetlib.iis.common.ClassPathResourceProvider;
+import eu.dnetlib.iis.common.schemas.ReportEntry;
+import eu.dnetlib.iis.common.spark.TestWithSharedSparkSession;
+import eu.dnetlib.iis.common.utils.AvroAssertTestUtil;
+import eu.dnetlib.iis.importer.auxiliary.schemas.DocumentContentUrl;
+
+/**
+ * @author mhorst
+ *
+ */
+public class HiveBasedDocumentContentUrlImporterJobTest extends TestWithSharedSparkSession {
+
+    @TempDir
+    public Path workingDir;
+    
+    private Path outputDir;
+    private Path outputReportDir;
+    
+    private HiveBasedDocumentContentUrlImporterJobTest() {
+        // enabling hive support in spark session instantiated for this set of tests
+        super(true);
+    }
+    
+    @BeforeEach
+    public void beforeEach() {
+        super.beforeEach();
```

Comment:
You're right. I didn't pay attention to this, just copy&pasted from some other test class. 

I guess it might be some leftover when the IIS test code base was switched from junit4 to junit5 as it seems the default behavior [has changed](https://stackoverflow.com/questions/48864716/before-beforeeach-inheritance-behaviour-change-junit4-junit5). Again, I didn't check this thoroughly so let's do the cleanup as a part of the dedicated #1365 task.